yaptide/celery/tasks.py

```diff
@@ -110,6 +114,30 @@ def pymchelper_output_to_json(estimators_dict: dict) -> dict:
     return result_dict
 
 
+def get_shieldhitlog(path: str) -> str:
+    """Proto version of returning errors -> returning shieldhit.log"""
+    try:
+        with open(path, 'r') as reader:
+            return reader.read()
+    except FileNotFoundError:
+        return "Shieldhitlog not found"
+
+
+def get_shield_input(path: str) -> dict:
```

Comment:
Rename to `simulation_input_files`,  add a docstring explaining what should be provided as input argument `path`.